Jenkinsfile.buildchain

```diff
@@ -143,13 +143,13 @@ pipeline {
             script {
                 if(findbugsFile) {
                     echo 'Findbugs reports ...'
-                    findbugs canComputeNew: false, defaultEncoding: '', excludePattern: '', healthy: '', includePattern: '', pattern: findbugsFile, unHealthy: ''
+                    recordIssues enabledForFailure: true, tool: spotBugs(), pattern: findbugsFile
```

Comment:
maybe you could extend the command from one line to more to be better readable ;)

Reply:
```suggestion
                    recordIssues (
                       enabledForFailure: true,
                       tool: findBugs(),
                       pattern: findbugsFile
                    )
```